freqtrade/optimize/hyperopt.py

```diff
@@ -504,66 +359,33 @@ def start(self) -> None:
         if self.has_space('buy'):
             self.analyze.populate_indicators = Hyperopt.populate_indicators  # type: ignore
         self.processed = self.tickerdata_to_dataframe(data)
+        self.exchange = None  # type: ignore
+        self.load_previous_results()
 
-        logger.info('Preparing Trials..')
-        signal.signal(signal.SIGINT, self.signal_handler)
-        # read trials file if we have one
-        if os.path.exists(self.trials_file) and os.path.getsize(self.trials_file) > 0:
-            self.trials = self.read_trials()
-
-            self.current_tries = len(self.trials.results)
-            self.total_tries += self.current_tries
-            logger.info(
-                'Continuing with trials. Current: %d, Total: %d',
-                self.current_tries,
-                self.total_tries
-            )
+        cpus = multiprocessing.cpu_count()
+        logger.info(f'Found {cpus}. Let\'s make them scream!')
```

Comment:
can we use `"` if a `'` is in the text ... 

not sure but most `logger.*()` calls seem to use `%s` formatting.
while outdated - it seems to provide consistancy